### DISCUSSION TITLE
Cocoapods dependency conflicts between Firebase and MLKit Barcode Scanning ?

### DISCUSSION THREAD

**Discussion by VicMoro3** [capawesome-team/capacitor-firebase#754] [Q&A]
It seems like I can't use both capawesome-team packages @capacitor-mlkit/barcode-scanning and @capacitor-firebase/crashlytics on the same project because of a dependency conflict between Firebase 11 and GoogleMLKit/BarcodeScanning 5.0 on iOS

```
[!] CocoaPods could not find compatible versions for pod "GoogleDataTransport":
  In Podfile:
    CapacitorFirebaseCrashlytics (from '../../node_modules/@capacitor-firebase/crashlytics') was resolved to 6.2.0, which depends on
      FirebaseCrashlytics (~> 11.0) was resolved to 11.4.0, which depends on
        GoogleDataTransport (~> 10.0)
 
    CapacitorMlkitBarcodeScanning (from '../../node_modules/@capacitor-mlkit/barcode-scanning') was resolved to 6.2.0, which depends on
      GoogleMLKit/BarcodeScanning (= 5.0.0) was resolved to 5.0.0, which depends on
        GoogleMLKit/MLKitCore (= 5.0.0) was resolved to 5.0.0, which depends on
          MLKitCommon (~> 10.0.0) was resolved to 10.0.0, which depends on
            GoogleDataTransport (~> 9.0)
```

This ressemble a past issue posted on the capawesome-team mlkit github : [https://github.com/capawesome-team/capacitor-mlkit/issues/23](https://github.com/capawesome-team/capacitor-mlkit/issues/23)

Has anyone encoutered the same issue ?

Does this require an issue to be created ? (and this would be on Google part right ?)

**Comment by robingenz** [comment]
Please install the latest version using the `next` tag:

```
npm install @capacitor-mlkit/barcode-scanning@next
```

For more information, see https://github.com/capawesome-team/capacitor-mlkit/issues/178.
